packages/teleport/src/services/alerts/alerts.tsx

```diff
@@ -17,6 +17,8 @@ limitations under the License.
 import api from 'teleport/services/api';
 import cfg from 'teleport/config';
 
+export const LinkLabel = 'teleport.internal/link';
```

Comment:
[nit]
Pascal case is reserved for react component types, typescript types/enums, and classes. Please use uppercase or camelCase
```suggestion
export const LINK_LABEL = 'teleport.internal/link';
```
I think we should enforce it by https://typescript-eslint.io/rules/naming-convention/